### ISSUE TITLE
Inconsistent in i18n Message Interpolation in `deny!` and `allowed_to?`

### ISSUE THREAD

**Issue by AlexeyMatskevich** [palkan/action_policy#276]
### Tell us about your environment

**Ruby Version:** 3.3.5

**Framework Version (Rails, whatever):** rails 8rc1

**Action Policy Version:** 0.7.1

**Reproduction Script:** Case is simple

### What did you do?
Create I18n for rule
```yaml
en:
  action_policy:
    policy:
      no_owner: "You are not the owner of %{model} with ID %{id}."
```
Apply rule
```ruby
  def owner?
    details[:model] = record.model_name.human
    details[:id] = record.id

    deny!(:no_owner) unless record.account_id == account.id
  end
```
### What did you expect to happen?
I expect that both the `deny!` and `allowed_to?` methods will provide the same behavior regarding i18n messages. Specifically, both methods should interpolate data from the details hash into message. I formed this impression based on the documentation found at [Action Policy - Detailed Reasons](https://actionpolicy.evilmartians.io/#/reasons?id=detailed-reasons).
### What actually happened?
Actually, I get this message:
```
"You are not the owner of %{model} with ID %{id}."
```
I see that this matches the code:
https://github.com/palkan/action_policy/blob/master/lib/action_policy/policy/reasons.rb#L223
where when using the `add` method `details` is not passed

Is this a bug or is it by design? Based on the documentation, I didn't expect any difference in behavior.
Thanks for the great gem!

**Comment by palkan** [comment]
> Is this a bug or is it by design?

Looks like a bug to me. Thanks for reporting! I will try to check and fix later this week.

**Comment by palkan** [comment]
Fixed in 0.7.2
